src/AspNet.Security.OAuth.Introspection/Events/BaseIntrospectionContext.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Http;
+
+namespace AspNet.Security.OAuth.Introspection
+{
+    public abstract class BaseIntrospectionContext : BaseControlContext
```

Comment:
I was under the assumption that `BaseControlContext` made sense because we were trying to control the flow... Additionally it already has `AuthenticationTicket` on it for convenience and the Skipped/Handled processing that could be used by the middleware to determine what processing to skip (such as default Audience validation)... but perhaps this stuff should only be added as-needed. What do you think?